flank-scripts/.gitignore

```diff
@@ -24,3 +24,4 @@ out/
 buildSrc/.gradle/
 buildSrc/build
 local.properties
+src/flank-scripts.properties
```

Comment:
I think that `flank-scripts.properties` should be placed under `flank-scripts/flank-scripts.properties`